src/components/SwitcherModal.vue

```diff
@@ -48,7 +48,7 @@
 <script setup lang="ts">
     import SpecimensGallery from '../components/SpecimensGallery.vue'
     import type {CardSpecimen} from '../components/SpecimensGallery.vue'
-    import seqMODULES from '../sequences/sequences'
+    import {seqMODULES} from '../sequences/sequences'
```

Comment:
I'm not sure this problem originates in this PR, but in workbench mode the switcher modal doesn't work.
![Screenshot from 2024-07-23 09-41-40](https://github.com/user-attachments/assets/6dfdc902-5aae-472f-b3b5-811166a439fc)

Reply:
That's because Grid is dead in the water from the original Paramable changes in the first PR. Fixing Grid up is pretty much the top priority for after the Delft PRs are in (and definitely before ui2->main).